jekyll/_cci2/high-availability.md

```diff
@@ -111,9 +111,9 @@ It is best practice to set up PostgreSQL 9.5 or later using Amazon Relational Da
 3. Confirm that all MongoDB and PostgreSQL containers have stopped by listing all running containers with the `docker ps` command.
 4. Download and run the export script using the commands below. The duration of the export operation depends on the amount of stored data.
      ```shell
-     wget https://s3.amazonaws.com/release-team/scripts/circleci-database-export
-     chmod +x circleci-database-export
-     ./circleci-database-export
+     wget https://s3.amazonaws.com/release-team/scripts/circle-database-export-2.0
+     chmod +x circleci-database-export-2.0
```

Comment:
Should this be circle-database-export-2.0 in all three places? (one is circle-database-export-2.0 and the other two are circleci-database-export-2.0). circleci- versus circle-